kong/cmd/utils/log.lua

```diff
@@ -24,6 +25,15 @@ function _M.set_lvl(lvl)
   end
 end
 
+function _M.disable()
+  old_lvl = log_lvl
```

Comment:
also, as a first line add `log(debug, "temporarily disabling logging")`, otherwise if the calls to disable/enable get unbalanced, it'll be very hard to find the offending piece of code.

Below add a similar 'reenabling log' message at the end.